_projects/2023/100483064/100483064.Rmd

```diff
@@ -0,0 +1,440 @@
+---
+title: "Graph: La evolución del despligue militar de EE.UU."
+description:
+  "This project will explain the process of replicating the graphic La evolución del despliegue militar de EE.UU. created by El Orden Mundial."
+categories: "2023"
+author: Clara Fernanda Martín Villaescusa
+date: "`r Sys.Date()`"
+output:
+  distill::distill_article:
+    self_contained: false
+    toc: true 
+---
+
+# REPLICATION
```

Comment:
The main title is already level 1. Headings insisde the post should start at level 2 (`##`).

Also, please put all headings in sentence case.